index.js

```diff
@@ -115,6 +115,11 @@ function vine(str) {
  * @returns {string|undefined}
  */
 function youtube(str) {
+
+	// remove time and feature
+	str = str.replace(/#t=.*/, "");
+	str = str.replace(/&feature=.*/, "");
```

Comment:
The last part of this regex looks a bit destructive (or greedy) in the sense that it will try to remove **everything** after the `feature=`, which might be unsafe, since we have any other routines that need to search for url parameters (see some of the functions below this one for examples like `v` or `vi`). So you would want to tighten up this RegExp and add some unit tests to cover all these edge cases.

Reply:
Also, it's not clear to me what value is provided by removing these specific url parameters. Can you elaborate @mikkopori?

Reply:
Nevermind, I see the value now that I ran your tests, without your implementation 

![image](https://user-images.githubusercontent.com/5614571/98437626-e6fdf480-20e3-11eb-9b57-97148930b261.png)

Reply:
But this RegEx is too tight. It will fail other url patterns that include `feature` like: 

```
http://www.youtube.com/attribution_link?a=fF1CWYwxCQ4&feature=em-uploademail&u=/watch?v=ABC12305
```

![image](https://user-images.githubusercontent.com/5614571/98437772-50323780-20e5-11eb-976a-64083809577e.png)

Reply:
It might be safer to remove the feature parameter after the id is extracted:

**Before**:

```
G-3YxlZIhus&feature=channel
```

**After**:

```
G-3YxlZIhus
```